test_aeo/dsgrid_project/datasets/growth_rate/Commercial_End_Use_Growth_Factors/dataset.toml

```diff
@@ -1,26 +1,30 @@
-data_source = "AEO2021 Reference Case"
+dataset_type = "growth_rate"
+data_source = "NEMS Commercial Module"
 dataset_id = "aeo2021_ref_com_energy_end_use_growth_factors"
 data_schema_type = "one_table"
-dataset_type = "benchmark" # <--- ?
-path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Use Growth Factors/data/" # <---
+path = "/Users/lliu2/Documents/GitHub/dsgrid-project-StandardScenarios/dsgrid_project/datasets/benchmark/aeo2021/reference/commercial/End Use Growth Factors/data/"
```

Comment:
Should this path be relative to the .toml file (or some other key path) instead of pointing to your machine?

Reply:
It gets replaced in the test code with a path to a temp dir where the data is copied.